### DISCUSSION TITLE
How can I make the background transparent ?

### DISCUSSION THREAD

**Discussion by AnkurAlpha** [lukas-reineke/indent-blankline.nvim#786] [Q&A]
how to make the background of it look transparent , currently it looks like this : 
![2023-12-05T19:36:02,621216739+05:30](https://github.com/lukas-reineke/indent-blankline.nvim/assets/132189303/31f4a2e2-3d5d-4a71-b4e2-86a1d9ed37da)
I want to make the background in it transparent . 
In config I have only this : 
~~~
require("ibl").setup()
~~~

Any help would be appriciated ,
thanks in advance :smile: !!!

**Comment by lukas-reineke** [comment]
Take a look at `:help ibl.config.whitespace.highlight` and `:help ibl.config.indent.highlight`

**Reply by AnkurAlpha** [reply]
shouldn't **remove_blankline_trail = true** be enough for this ?
~~~
local highlight = {
    "Whitespace",
    "NonText",
}
require("ibl").setup {
    indent = { highlight = highlight, },
    whitespace = {
        highlight = "Whitespace",
        remove_blankline_trail = true,
    },
    scope = { enabled = false },
}
~~~
![2023-12-06T13:52:07,739072947+05:30](https://github.com/lukas-reineke/indent-blankline.nvim/assets/132189303/aa874281-23da-484a-97d4-6962fcf0c27e)
I am not getting the background transparent :frowning_face:

**Reply by lukas-reineke** [reply]
`remove_blankline_trail` Is unrelated to this.
You just need to set a highlight group without background color.
